docs/udf.rst

```diff
@@ -279,8 +279,50 @@ instead of the original digital number range (thousands):
 
 Illustration of data chunking in ``apply`` with a  UDF
 ========================================================
+Chunk_polygon can be used to apply a specific kind of process to spatial chunks of a data cube.   
 
-TODO
+
+In the following example, we repeat the method of rescaling a data cube using chunk_polygon.
+.. code-block:: python
+    :linenos:
+    :caption: UDF along with chunk_polygon 
+    :emphasize-lines: 14-25
+
+    import openeo
+
+    # Create connection to openEO back-end
+    connection = openeo.connect("...").authenticate_oidc()
+
+    # Load your data cube based on your prefernce
+
+    S2_cube = connection.load_collection(
+        "SENTINEL2_L2A",
+        temporal_extent = ["2022-06-04", "2022-08-04"],
+        bands = ["B02", "B03", "B04"]
+    )
+
+    # Create a UDF object from inline source code.
+    my_code = """
+    from openeo.udf import XarrayDataCube
+
+    def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
+        array = cube.get_array()
+        array.values = 0.0001* array.values
+        return cube
+    """
+    my_udf = lambda data: data.run_udf(udf=my_code,runtime='python')
+    
+    #Though a function to read JSON is shown here, you can replace it with your process.
+    aoi = read_json("./aoi/your_aoi.geojson")
```

Comment:
I think you should clarify what this geojson contains: just a simple polygon or a feature collection?